src/package.rs

```diff
@@ -1,39 +1,39 @@
 use std::collections::HashSet;
 
-use crate::sym::Sym;
+use crate::symbol::Symbol;
 
 #[derive(Clone, Debug)]
 pub struct Package {
-    pub name: Sym,
-    pub external_symbols: HashSet<Sym>,
+    pub name: Symbol,
+    pub external_symbols: HashSet<Symbol>,
 }
 
 pub const LURK_EXTERNAL_SYMBOL_NAMES: &[&str] = &[
-    "atom",
-    "begin",
-    "car",
-    "cdr",
-    "char",
-    "comm",
-    "commit",
-    "cons",
-    "current-env",
-    "emit",
-    "eval",
-    "eq",
-    "hide",
-    "if",
-    "lambda",
-    "let",
-    "letrec",
-    "nil",
-    "num",
-    "u64",
-    "open",
-    "quote",
-    "secret",
-    "strcons",
-    "t",
+    "ATOM",
+    "BEGIN",
+    "CAR",
+    "CDR",
+    "CHAR",
+    "COMM",
+    "COMMIT",
+    "CONS",
+    "CURRENT-ENV",
+    "EMIT",
+    "EVAL",
+    "EQ",
+    "HIDE",
+    "IF",
+    "LAMBDA",
+    "LET",
+    "LETREC",
+    "NIL",
+    "NUM",
+    "U64",
+    "OPEN",
+    "QUOTE",
+    "SECRET",
+    "STRCONS",
+    "T",
     "_",
```

Comment:
Aren't the built-in Lurk symbols now defined to be lower-case? They used to be defined to be upper-case but were converted on reading. Now they aren't converted and shouldn't be. So having this section manually upcase them violates my expectations.